versioneer.py

```diff
@@ -1357,7 +1425,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, runner=run_command):
     return pieces
 
 
-def do_vcs_install(versionfile_source, ipy):
+def do_vcs_install(versionfile_source: str, ipy: Optional[str]) -> None:
     """Git-specific installation logic for Versioneer.
```

Comment:
One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: `:param` `:return`)